### DISCUSSION TITLE
Welcome to Libra Discussions!

### DISCUSSION THREAD

**Discussion by alexvakimov** [Quantum-Dynamics-Hub/libra-code#114] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by alexvakimov** [comment]
Hi all, 
Just to give an example of your first post on the discussions here (see the suggestions in the very first one), let me start. 

👋 My name is Alexey Akimov.  👨‍🏫 I'm an Assistant Professor at the University at Buffalo (UB) 

My [group](https://akimovlab.github.io/) works on various topics in quantum-classical and quantum nonadiabatic dynamics, including the development of new methods for excited states dynamics simulation in periodic and nanoscale models of materials.  

I am the principal developer and maintainer of this (Libra) code, so it is likely you'll see my posts more frequently here.  Also, @bsmith24,  @WeiLiweili, @MohammadShakiba, and @storytemen have been helping a lot recently, and @Pulokesh and @kosukesato in the past. Guys, please introduce yourself, and let's keep the discussion going.  

P.S. you can use Markdown styling here, emoticons, and referencing someone (I just tried a few of those things).

**Reply by MohammadShakiba** [reply]
Hello everyone, 👋 

Thank you very much Alexey for starting this 🙏. Let me introduce myself. My name is Mohammad Shakiba and I have recently completed a master's in Nanomaterials Engineering at the Shahid Bahonar University of Kerman in Iran. I have experience in coding with Python, MATLAB, and some C/C++. With Libra you can use different software packages and methodologies available in them to do nonadiabatic dynamics in different bases. Recently, we have implemented a new methodology in Libra which is able to do the nonadiabatic dynamics both in [single-particle and many-body](https://pubs.acs.org/doi/10.1021/acs.jctc.0c01009) (LR-TD-DFT) bases using CP2K, Gaussian, and DFTB+ software packages. 

I'll be happy if I can answer your questions about using Libra. 😉😊
